cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -38,6 +41,7 @@ def __init__(self, ignore_labels=False):
         self.schema_def = dict()
         self.schema_version: str = None
         self.ignore_labels = ignore_labels
+        self.visium_and_is_single_true_matrix_size = VISIUM_AND_IS_SINGLE_TRUE_MATRIX_SIZE
```

Comment:
nit: in the future you can patch `VISIUM_AND_IS_SINGLE_TRUE_MATRIX_SIZE` in the tests to avoid having this class variable. Just one less duplicated variable.